source/app/helpers/sessions_helper.rb

```diff
@@ -1,2 +1,18 @@
 module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
+  end
+
+  def is_admin?
+    return false if current_user.nil?
+    current_user.user_type == 'admin'
```

Comment:
I think it would look a little better if this line were: `current_user.admin?`. Wanna implement that method?